src/main/scala/Ch2.scala

```diff
@@ -0,0 +1,30 @@
+package swscala
+
+
+object Exercises1 {
+
+  val zeroToNine = (0 to 9)
+
+  // Problem 1
+  lazy val p1: Seq[(Int, Int)] = zeroToNine.flatMap(i => zeroToNine.map(j => (i, j)))
+    .filter{case (i: Int, j: Int) => i + 4 * j  > i * j}
```

Comment:
Is it necessary to write type annotations `i: Int, j:Int` and in other places? Would just `case (i, j) => ...` work here?

Reply:
I'll try it out!